src/test/unit/controllers/StatusDescriptionController.test.ts

```diff
@@ -0,0 +1,25 @@
+import sinon from 'sinon';
+import { Request, Response } from 'express';
+import StatusDescriptionController from '../../../main/controllers/StatusDescriptionController';
+
+describe('Status Description Controller', () => {
+  it('should render the alphabetical search page', () =>  {
+    const statusDescriptionController = new StatusDescriptionController();
+
+    const response = {
+      render: function() {return '';},
+      get: function() {return 'test test;script-src test';},
```

Comment:
whats with all the return params in `get` i dont see you check for them anywhere?